### ISSUE TITLE
Not compatible with RN 0.60

### ISSUE THREAD

**Issue by chrisspankroy** [Estimote/react-native-proximity#46]
After upgrading a project to RN 0.60, `pod install` fails, citing `react-native-proximity` as the cause:
```bash
[!] use_native_modules! skipped the react-native dependency '@estimote/react-native-proximity'. No podspec file was found.
    - Check to see if there is an updated version that contains the necessary podspec
    file
    - Contact the library maintainers or send them a PR to add a podspec. The
    react-native-webview podspec is a good example of a package.json driven podspec. See
    https://github.com/react-native-community/react-native-webview/blob/master/react-native-webview.podspec
    - If necessary, you can disable autolinking for the dependency and link it manually.
    See
    https://github.com/react-native-community/cli/blob/master/docs/autolinking.md#how-can-i-disable-autolinking-for-unsupported-library
```
If you run `pod install` using React Native version 0.60.0, you'll get a generic error `TypeError - no implicit conversion of nil into String`
If you run `pod install` using React Native's `master` branch, you'll get this specific error (they have added better logging after they released version 0.60.0)

https://github.com/react-native-community/cli/issues/506

Basically a podspec needs to be added

**Comment by heypiotr** [comment]
Thanks for the heads up! This is actually a really cool and exciting change, because it means I can now make the Estimote iOS Proximity SDK the dependency of react-native-proximity (via the new podspec support), which will simplify the installation and updates. I'll try to get it implemented this week.

**Comment by heypiotr** [comment]
I've pushed 0.6.0\* with support for RN 0.60. Let me know if it worked for you.

\* a nice, accidental match for 0.60 support :wink:

**Comment by chrisspankroy** [comment]
Awesome, thanks! It seems to work for me! Now I get to go track down another build issue 😕 (Not related to this library)

**Comment by heypiotr** [comment]
The woes of updating and living on the cutting edge :wink:

Glad we've managed to at least take one problem off your hands!
